.travis.yml

```diff
@@ -22,3 +22,26 @@ jobs:
         - docker rm -f pgbuild
     - stage: test
       script: PGTEST_TMPDIR=$TRAVIS_BUILD_DIR bash -x ./scripts/test_updates.sh
+
+    - stage: test
```

Comment:
I don't understand why we use stages but the same name for each stage? Also, stages are run in serial fashion instead of parallel so it slows down the build. Maybe I am missing something w.r.t. stages?